datahub-web-react/src/app/entity/glossaryTerm/preview/Preview.tsx

```diff
@@ -35,6 +37,7 @@ export const Preview = ({
             typeIcon={entityRegistry.getIcon(EntityType.GlossaryTerm, 14, IconStyleType.ACCENT)}
             deprecation={deprecation}
             parentNodes={parentNodes}
+            glossaryAssociatedDomain={domain}
```

Comment:
can you just reuse the existing `domain` prop on `DefaultPreviewCard` instead of creating a new `glossaryAssociatedDomain` prop?